frontend/content/posts/rds-native-backup-and-restore/index.yml

```diff
@@ -0,0 +1,11 @@
+title: "RDS SQL Server - Native Backup and Restore in CDK"
+excerpt: "Today we are going to build a popular solution that uses S3 to host your SQL backup file and import this into your new RDS database."
+banner: rds.jpg
+authorIds:
+  - edwin-radtke
+url: https://edwin.hashnode.dev/rds-sql-server-native-backup-and-restore-in-cdk
+createdAt: "2020-09-20 19:00"
```

Comment:
I added the date where I actually added it here to the repo. Sort of like a timestamp in a db, so that newly added content gets sorted to the top. Up to you if you wanna adjust this or not

Reply:
Yes thanks, I wanted it to be lower down since I feel there are a lot better posts right now. Especially on launch.👍

Reply:
That's too modest of you ;) But yeah, I'll merge it as it is 👍